src/SLCore.Listeners.UnitTests/Logging/LoggerListenerTests.cs

```diff
@@ -40,12 +40,12 @@ public void Mef_CheckIsSingleton()
         }
 
         [TestMethod]
-        [DataRow(LogLevel.ERROR, true)]
-        [DataRow(LogLevel.WARN, true)]
-        [DataRow(LogLevel.INFO, false)]
-        [DataRow(LogLevel.TRACE, false)]
-        [DataRow(LogLevel.DEBUG, false)]
-        public void Log_LogsOnlyErrorAndWarning(LogLevel logLevel, bool logs)
+        //[DataRow(LogLevel.ERROR, false)]
+        //[DataRow(LogLevel.WARN, false)]
+        //[DataRow(LogLevel.INFO, true)]
+        //[DataRow(LogLevel.TRACE, true)]
+        [DataRow(LogLevel.DEBUG, true)]
+        public void Log_LogsOnlyErrorAndWarning(LogLevel logLevel, bool verboseLogs)
```

Comment:
Changing from `bool logs` to `bool verboseLogs,` also changes the scope of the test.

Basically, it was checking if it logs only on `ERROR` and `WARN`, so on the `INFO`, `TRACE` and `DEBUG` I don't expect to see anything on the logs.

However, now it always logs something (either a regular log or a verbose log), which is confusing. I am not sure whether the test's name is the issue or if this should split into 2 different tests or if it's the intended TestLogger's behavior. 🤔

Reply:
Old test did not make sense, for info, trace and debug it should call the logverbose and based on log level it's shown or not but on the loggerlistener test this is not important. I've changed the name of the test to remove confusion.